go-controller/pkg/util/ovs.go

```diff
@@ -328,6 +328,44 @@ func GetOVSOfPort(args ...string) (string, string, error) {
 	return stdout, stderr, err
 }
 
+func GetDatapathType(bridge string) (string, error) {
+	br_type, err := GetOvsEntry("bridge", "br-int", "datapath_type", "")
+	if err != nil {
+		return "", err
+	}
+	return br_type, nil
+}
+
+// GetOvsEntry queries the OVS-DB using ovs-vsctl and returns
+// the requested entries.
+func GetOvsEntry(table, record, column, key string) (string, error) {
+	args := []string{"--if-exists", "get", table, record}
+	if key != "" {
+		args = append(args, fmt.Sprintf("%s:%s", column, key))
+	} else {
+		args = append(args, column)
+	}
+	stdout, stderr, err := RunOVSVsctl(args...)
+	if err != nil {
+		return "", fmt.Errorf("failed to run 'ovs-vsctl %s': %v\n  %q",
```

Comment:
don't use `\n` in the log messages.. it will confuse the Log aggregation platform .. and hard to debug when we are grepping for messages in the log file ..  just replace `\n` with ':'

Reply:
looks like there are two spaces after '\n` remove one space.

Reply:
ack